src/tests/PaginationArrow.spec.js

```diff
@@ -34,31 +32,10 @@ describe('The Pagination Arrow component', function () {
       onTrigger: clickTrigger,
     });
     expect(this.renderedComponent).toEqual(
-      <div className='my-class' onClick={clickTrigger}>
-        <div className='PaginationArrowIcon' />
+      <div bemBlock='DateRangePicker' className='DateRangePicker__PaginationArrow DateRangePicker__PaginationArrow--next' onClick={clickTrigger} >
+        <div className='DateRangePicker__PaginationArrowIcon DateRangePicker__PaginationArrowIcon--next' />
       </div>
     );
   });
 
```

Comment:
The whitespace at the end the `it` block encapsulating `describe` block can be nuked IMO; However you're right, there is no linting rule for this. Happy for you to punt on this.